R/migraph-package.R

```diff
@@ -0,0 +1,7 @@
+#' @importFrom rlang `%||%`
```

Comment:
Added here because of https://usethis.r-lib.org/reference/use_import_from.html (it could be elsewhere of course)